commit-style.md

```diff
@@ -8,9 +8,11 @@
 5. Capitalize the subject line
 6. Do not end the subject line with a period
 7. Use the body for explain what and why vs. how.
+8. If you have made a mistake in your commit message, you can fix it using the commad git commit --amend
```

Comment:
Also, you can use "Ammend" button in SourceTree git client.
And a typo here - commaNd